pages/content/amp-dev/documentation/guides-and-tutorials/learn/spec/release-schedule.md

```diff
@@ -35,7 +35,7 @@ A new release of AMP is pushed to all AMP pages every week on Tuesday. **Once a
 
 The AMP runtime and extensions are provided through a variety of different _release channels_. Each channel serves a purpose for developers and for the AMP HTML Project itself. See the [release cadence section](#release-cadence) for a more detailed breakdown of how and when code from the [`ampproject/amphtml`](https://github.com/ampproject/amphtml) repository makes it into release builds.
 
-To determine if a PR has been included in any of the following release channels, look for the GitHub labels _PR Use: In Canary_, _PR Use: In Production_, or _PR Use: In LTS_ (see the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release) for more details).
+To determine if a PR has been included in any of the following release channels, look for the GitHub labels _PR Use: In Canary_, _PR Use: In Production_, or _PR Use: In LTS_ (see the section on [determining if your change is in a release](#determining-if-your-change-is-in-a-release) for more details).
```

Comment:
Are changes to this file a side effect of `gulp bootstrap`? If so let's remove since it is not relevant to the larger purpose of the PR.

Reply:
Ok, it is removed.